### DISCUSSION TITLE
LCD keeps looping and won't come up after updating to v0.2.9-39-gc0ffa41

### DISCUSSION THREAD

**Discussion by agorafalaht** [KlipperScreen/KlipperScreen#808] [Q&A]
my lcd is looping after the update, I've already reinstalled KlipperScreen, I've done a new update and nothing, the display has some texts restarting directly, the dashboard is ok, and the printer worked.

**Comment by alfrix** [comment]
when you have issues please provide the logfiles (sorry, i rarely check discussions)
closing because i assume at this point that you solved it
